.circleci/config.yml

```diff
@@ -133,12 +133,12 @@ jobs:
     description: << parameters.path >> unit test
     steps:
       - *restore_repo
-      - restore_plugin_pods:
-          prefix: << parameters.path >>
+#      - restore_plugin_pods:
+#          prefix: << parameters.path >>
```

Comment:
Unless there's a compelling reason to keep these commented, remove it altogether

Reply:
i believe @iartemiev was testing a fix for the pipeline issue we were experiencing

Reply:
change has been reverted